storage/dynamodb/user_storage.go

```diff
@@ -502,19 +503,22 @@ func (us *UserStorage) FetchUsers(filterString string, skip, limit int) ([]model
 	result, err := us.db.C.Scan(scanInput)
 	if err != nil {
 		log.Println("Error querying for users:", err)
-		return nil, ErrorInternalError
+		return []model.User{}, 0, ErrorInternalError
 	}
 
 	users := make([]model.User, len(result.Items))
 	for i := 0; i < len(result.Items); i++ {
+		if i < skip {
+			continue // TODO: use internal pagination mechanism
```

Comment:
Probably there's still something to do, in case you're done remove that comment, please.

Reply:
It's a long-term TODO, because:
1) it is not such a trivial task to comply DynamoDB pagination mechanics with other ones, like MongoDB, seamlessly for the API layer;
2) this pull request is already massive enough;
3) this is not a critical issue for the time being.